package.json

```diff
@@ -62,6 +62,10 @@
 			"require": "./compat/server.js",
 			"import": "./compat/server.mjs"
 		},
+		"./compat/scheduler": {
+			"require": "./compat/dist/scheduler.js",
```

Comment:
The CI fails because this file is not present there. Maybe we need to build first?

Reply:
that's an option, yeah. We don't currently provide the `scheduler` package, and it's only ever CJS. So I was thinking maybe we move it from `compat/src/scheduler.js` to `compat/scheduler.{mjs,js}` just like we do for `compat/server`?

Reply:
sounds good to me 👍